apps/site/README.md

```diff
@@ -14,7 +14,7 @@ This folder contains the code for [blockprotocol.org](https://blockprotocol.org)
     - `HASHING_SECRET`: the secret used to hash API keys
     - `SESSION_SECRET`: the secret used to sign the session ID cookie
     - `MONGODB_URI`: the URL where the mongo developer db instance is hosted (for example at `mongodb://root:password@localhost:27017/`)
-    - `MONGODB_DB_NAME`: the name of the database (for example `local`)
+    - `MONGODB_DB_NAME`: the name of the database (for example `development-database`)
```

Comment:
See [this slack thread](https://hashintel.slack.com/archives/C02LG39FJAU/p1677599645272769) for reasoning behind this.